src/parquet/column_reader.cc

```diff
@@ -212,6 +121,9 @@ bool ColumnReader::ReadNewPage() {
       // Read a data page.
       num_buffered_values_ = current_page_header_.data_page_header.num_values;
```

Comment:
Can we not pass the maximum def/rep level as an argument in the API?
We do not need this for DataPageV2. Can we have a field inside the ColumnReader instead ?